### PR TITLE
SONAR-9565 Move quality gates link at org level

### PR DESCRIPTION
**Pull request by gregaubert**

### CODE REVIEW

**Comment by gregaubert** [comment]
@stas-vilchik Please review only last commit

File: server/sonar-web/src/main/js/apps/overview/meta/Meta.js [hunk 1]
```diff
@@ -56,7 +56,11 @@ const Meta = ({ component, history, measures, areThereCustomOrganizations, route
 
       {isProject && <MetaTags component={component} />}
 
-      {shouldShowQualityGate && <MetaQualityGate gate={qualityGate} />}
+      {shouldShowQualityGate &&
+        <MetaQualityGate
+          gate={qualityGate}
+          organization={shouldShowOrganizationKey && component.organization}
```
Comment (stas-vilchik) [inline server/sonar-web/src/main/js/apps/overview/meta/Meta.js]: `shouldShowOrganizationKey` should be probably renamed

File: server/sonar-web/src/main/js/apps/quality-gates/components/Details.js [hunk 2]
```diff
@@ -85,14 +86,14 @@ export default class Details extends Component {
...
+    const { qualityGate, onDelete, organization } = this.props;
     const { router } = this.context;
-
+    const urlPrefix = organization ? '/organizations/' + organization.key : '';
     new DeleteView({
       qualityGate,
       onDelete: qualityGate => {
         onDelete(qualityGate);
-        router.replace('/quality_gates');
+        router.replace(urlPrefix + '/quality_gates');
```
Comment (stas-vilchik) [inline server/sonar-web/src/main/js/apps/quality-gates/components/Details.js]: what about creating `getQualityGatesUrl` (with "s") the same way as `getQualityGateUrl`?

Comment (gregaubert) [inline server/sonar-web/src/main/js/apps/quality-gates/components/Details.js]: Good idea! :+1:

File: server/sonar-web/src/main/js/apps/overview/meta/Meta.js (hunk 1 above)
Comment (gregaubert) [inline server/sonar-web/src/main/js/apps/overview/meta/Meta.js]: Renamed to `hasOrganization`

**Pull request merged**